sdks/python/apache_beam/yaml/standard_providers.yaml

```diff
@@ -0,0 +1,22 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- type: 'beamJar'
+  gradle_target: 'sdks:java:extensions:sql:expansion-service:shadowJar'
+  version: '2.43.0'
```

Comment:
Are we going to evolve this version with Beam releases or will these be fixed till we upgrade ?

Reply:
Good question. The versioning story hasn't been figured out yet--for the builtin ones we probably want to track the Beam release automatically.

Reply:
For now, I'm just adding a magic string `BEAM_VERSION` that'll use the current version of this beam install.